app/Services/ArticleService.php

```diff
@@ -37,6 +37,10 @@ public function getCategorizedArticlesByMagazine($magazineId)
         // For now, simply sorting categories alphabetically.
         ksort($categories);
 
+        foreach ($categories as $index => $category) {
+            $categories[$index] = collect($category)->sortByDesc('created_at');
+        }
```

Comment:
@kapilsharma I've found a better approach to implement this. I just used sortByDesc() in line 23 on the collection $articles . Resultant magazine has all the articles sorted. Should I push that code to this branch?